.github/workflows/ci.yaml

```diff
@@ -56,7 +62,6 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           python -m pip install -r requirements/docs.txt
-          python -m pip install .
```

Comment:
You do need to install the package itself for Sphinx to build it's documentation, so re-add this installation.

Reply:
yes added